File: JS/problem4.js
var start = 100;
var end = 999;
var winners = [];
var temp=[];

for (var i= start; i <= end; i++){
  for (var j = end; j >= start; j--){
   temp = (i * j)
   if (temp.toString().split('').reverse().join('') === temp.toString()){
   
   winners.push(temp);
     
    }
   
   }
}
console.log(winners.sort(function(a, b){return a-b}).pop())

File: Python/problem4.py
start = 100
end = 999
winners = []

for i in range(100, 999):
    for j in range(100, 999):
        temp = (i * j)
        if str(temp) == str(temp)[::-1]:
            winners.append(temp)
winners.sort()
print winners[-1]

